datasets/sentinel-1-rtc/dataset.yaml

```diff
@@ -17,11 +18,12 @@ collections:
     template: ${{ local.path(./collection) }}
     class: s1rtc:S1RTCCollection
     asset_storage:
-      - uri: blob://sentinel1euwestrtc/sentinel1-grd-rtc-stac
+      - uri: blob://sentinel1euwestrtc/sentinel1-grd-rtc-stac/GRD/2023/5/  # testing
```

Comment:
LMK if you're able to handle setting these to values appropriate for running daily.

For goes-glm, we do have an optimization to only list assets under the year `2023`. We can do that with an `arg` in the `dataset.yaml` and have the caller pass in `--arg extra-prefix "2023/".